source/includes/stackpath/_instances.md

```diff
@@ -156,6 +156,7 @@ curl -X POST \
     "created": "2021-04-20T20:58:59.952881-04:00",
     "result": {
       "clientId": "1e15e2050fe44771c5dd82dcf07c",
+      "cientSecret": "c3507b64a1cfe8a973d7aa234d7581dd99c732e2c1184f9ac90f803b83ad1887",
```

Comment:
We don't give a description of the clientId, clientSecret and sshCommand?

Reply:
Added them [here](https://github.com/cloudops/cloudmc-api-docs/pull/556/commits/9b01b208e3e9361b8a2ed499387fe3edfaa22b0b), good catch @fyrnaga